flavors/standard-Exasol-8-cuda-ml/flavor_base/conda_deps/_activate_current_env.sh

```diff
@@ -0,0 +1,15 @@
+# This script should never be called directly, only sourced:
```

Comment:
We probably should move this to ext and reuse it in all conda flavors, but this can be done in a different PR